core-web/libs/block-editor/src/lib/extensions/image-tabview-form/image-tabview-form.component.ts

```diff
@@ -0,0 +1,154 @@
+import { Component, OnInit, Input, ViewChild, ElementRef } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { BehaviorSubject, merge } from 'rxjs';
+import { debounceTime, take, map, throttleTime, mergeMap, tap } from 'rxjs/operators';
+import { Observable } from 'rxjs/internal/Observable';
+
+import { DotCMSContentlet } from '@dotcms/dotcms-models';
+
+import { DotLanguageService, Languages } from '../../shared';
+
+import {
+    ESOrderDirection,
+    queryEsParams,
+    SearchService
+} from '../../shared/services/search/search.service';
+
+@Component({
+    selector: 'dot-image-tabview-form',
+    templateUrl: './image-tabview-form.component.html',
+    styleUrls: ['./image-tabview-form.component.scss']
+})
+export class ImageTabviewFormComponent implements OnInit {
+    @ViewChild('inputSearch') inputSearch!: ElementRef;
+
+    @Input() languageId = 1;
```

Comment:
1 Could be a Enum LANGUAGES.ENGLISH ?

Reply:
Sure!

Reply:
I used this variables we have: `DEFAULT_LANG_ID`.